src/goGutter.ts

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import vscode = require('vscode');
+
+export let coveredGutter;
+export let uncoveredGutter;
+
+export function initGutterDecorators(ctx: vscode.ExtensionContext) {
```

Comment:
Fixed in next commit. You're right, it is cleaner to init the gutter in goCover.ts

The thing about calling from goMain.ts is that  I need an instance of `vscode.ExtensionContext` in order to access the absolute path of the images folder (gutter colors are actually SVGs rendered in the gutter) :

`gutterIconPath: ctx.asAbsolutePath('images/gutter-green.svg')`
or
`gutterIconPath: ctx.asAbsolutePath('images/gutter-red.svg')`

I did not find another way to access the `vscode.ExtensionContext` from goCover.ts (or any other file other than goMain.ts)